buildSrc/android-library-base.gradle

```diff
@@ -0,0 +1,30 @@
+apply plugin: 'com.android.library'
+
+
+android {
+    compileSdkVersion Versions.compile_sdk
+    buildToolsVersion Versions.build_tools
+
+    defaultConfig {
+        minSdkVersion Versions.min_sdk
+        targetSdkVersion Versions.target_sdk
+        versionCode 1
```

Comment:
It's a bit weird to share versionCode and versionName, maybe simply remove this.